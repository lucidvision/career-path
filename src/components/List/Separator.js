import React from 'react'
import { StyleSheet, View } from 'react-native'

export default function Separator () {
  return (
    <View style={styles.separator} />
  )
}

const styles = StyleSheet.create({
  separator: {
    backgroundColor: 'black',
    height: StyleSheet.hairlineWidth,
    marginHorizontal: 20
  }
})
