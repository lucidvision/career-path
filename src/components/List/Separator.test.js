import React from 'react'
import { shallow } from 'enzyme'
import Separator from './Separator'

describe('Separator', () => {
  const component = shallow(<Separator />)

  it('renders correctly', () => {
    expect(component).toMatchSnapshot()
  })
})
