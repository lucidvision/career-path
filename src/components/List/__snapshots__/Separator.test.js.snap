// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Separator renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "black",
      "height": 0.5,
      "marginHorizontal": 20,
    }
  }
/>
`;
